lib/pangea/pages/class_settings/p_class_widgets/room_capacity_button.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:fluffychat/pages/chat_details/chat_details.dart';
+import 'package:fluffychat/pangea/extensions/pangea_room_extension/pangea_room_extension.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:future_loading_dialog/future_loading_dialog.dart';
+import 'package:matrix/matrix.dart';
+
+class RoomCapacityButton extends StatefulWidget {
+  final Room? room;
+  final ChatDetailsController? controller;
+  const RoomCapacityButton({
+    super.key,
+    this.room,
+    this.controller,
+  });
+
+  @override
+  RoomCapacityButtonState createState() => RoomCapacityButtonState();
+}
+
+class RoomCapacityButtonState extends State<RoomCapacityButton> {
+  Room? room;
+  ChatDetailsController? controller;
+  String? capacity;
+  String? nonAdmins;
```

Comment:
Why should this variable be a string instead of an int? Not sure of the advantage to that. If there is one, can you add a comment in the code file explaining it?